cpmonitor/templates/base.html

```diff
@@ -107,9 +107,31 @@
       </div>
       <footer id="site-footer">
         <div class="site-footer-inner">
-          <p>
-            Copyright © <a href="https://localzero.net/">LocalZero.net</a>
-          </p>
+          <div>
+            <p>
+              Die Software hinter LocalZero Monitoring ist unter der
+              <a href="https://github.com/GermanZero-de/klimaschutzmonitor/blob/main/LICENSE">MIT License</a>
```

Comment:
Add target="_blank"

Reply:
Opening links in new tabs seems to be surprisingly controversial now that I look into it. I' think Id rather leave it as it is and leave it to the user to open it in a new tab with ctrl-click or middle mouse button if they prefer.